jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/HttpStream.java

```diff
@@ -19,20 +19,20 @@
 import org.eclipse.jetty.http.HttpFields;
 import org.eclipse.jetty.http.MetaData;
 import org.eclipse.jetty.io.Connection;
+import org.eclipse.jetty.io.Content;
 import org.eclipse.jetty.util.Callback;
 
 public interface HttpStream extends Callback
 {
     /**
      * @return an ID unique within the lifetime scope of the associated protocol connection.
      * This may be a protocol ID (eg HTTP/2 stream ID) or it may be unrelated to the protocol.
-     * @see HttpStream#getId();
      */
     String getId();
 
     long getNanoTimeStamp();
 
-    Content readContent();
+    Content.Chunk readContent();
 
     void demandContent(); // Calls back on Channel#onDataAvailable
```

Comment:
Now that this class is dealing with `Chunk`s instead of `Content`s this method should be renamed. And the comment points to an invalid method name that should be fixed too. :-)